editor/components/inserter/test/menu.js

```diff
@@ -90,256 +91,240 @@ const items = [
 	sharedItem,
 ];
 
-describe( 'InserterMenu', () => {
-	// NOTE: Due to https://github.com/airbnb/enzyme/issues/1174, some of the selectors passed through to
-	// wrapper.find have had to be strengthened (and the filterWhere strengthened also), otherwise two
-	// results would be returned even though only one was in the DOM.
+const defaultProps = {
+	position: 'top center',
+	instanceID: 1,
+	items: items,
+	debouncedSpeak: noop,
+	fetchSharedBlocks: noop,
+	setTimeout: noop,
+};
+
+let wrapper;
+
+const setWrapperForProps = ( propOverrides ) => {
```

Comment:
Honest curiosity: Why arrow functions vs. the `function` keyword?

Reply:
habit mostly :)